src/screens/WebsocketResearchScreen/SampleChatScreen.tsx

```diff
@@ -4,75 +4,87 @@
 import * as React from 'react';
 import {Dimensions, FlatList, KeyboardAvoidingView, Platform, StyleSheet, View} from 'react-native';
 
-import AnonymousChatHeader from '../../components/AnonymousChat/AnonymousChatHeader';
 import AnonymousInputMessage from '../../components/Chat/AnonymousInputMessage';
 import BaseChatItem from '../../components/AnonymousChat/BaseChatItem';
-import useAnonymousChatScreenHook from '../../hooks/screen/useAnonymousChatScreenHook';
+import ChatDetailHeader from '../../components/AnonymousChat/ChatDetailHeader';
+import useChatScreenHook from '../../hooks/screen/useChatScreenHook';
+import {ANONYMOUS} from '../../hooks/core/constant';
 import {colors} from '../../utils/colors';
 
 const {height} = Dimensions.get('window');
 
+export const styles = StyleSheet.create({
+  keyboardAvoidingView: {
+    flex: 1,
+    backgroundColor: colors.white
+  },
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    height: height - 85
+  },
+  chatContainer: {
+    display: 'flex',
+    height: '100%',
+    marginBottom: 72
+  },
+  inputContainer: {
+    backgroundColor: colors.white,
+    position: 'absolute',
+    bottom: 0,
+    // height: 50,
+    left: 0,
+    right: 0,
+    zIndex: 100,
+    padding: 8,
+    paddingBottom: 16,
+    borderTopColor: colors.lightgrey,
+    borderTopWidth: 1
+  }
+});
+
 const SampleChatScreen = () => {
+  const flatlistRef = React.useRef<FlatList>();
   const {selectedChannel, chats, goBackFromChatScreen, goToChatInfoScreen, sendChat} =
-    useAnonymousChatScreenHook();
-  const styles = StyleSheet.create({
-    keyboardAvoidingView: {
-      flex: 1,
-      backgroundColor: colors.white
-    },
-    container: {
-      display: 'flex',
-      flexDirection: 'column',
-      height: height - 85
-    },
-    chatContainer: {
-      display: 'flex',
-      height: '100%',
-      marginBottom: 72
-    },
-    inputContainer: {
-      backgroundColor: colors.white,
-      position: 'absolute',
-      bottom: 0,
-      // height: 50,
-      left: 0,
-      right: 0,
-      zIndex: 100,
-      padding: 8,
-      paddingBottom: 16,
-      borderTopColor: colors.lightgrey,
-      borderTopWidth: 1
-    }
-  });
+    useChatScreenHook(ANONYMOUS);
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useChatScreenHook` hook is now being used with the `ANONYMOUS` constant passed as an argument. This is a good change as it makes the code more flexible and adaptable to different chat types.

```diff
- const {selectedChannel, chats, goBackFromChatScreen, goToChatInfoScreen, sendChat} = useAnonymousChatScreenHook();
+ const {selectedChannel, chats, goBackFromChatScreen, goToChatInfoScreen, sendChat} = useChatScreenHook(ANONYMOUS);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->